### PR TITLE
fix: updating scheduled deliveries email layout

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4646 

### Description:
https://user-images.githubusercontent.com/67699259/223174869-5282ddf0-f601-446c-8534-6b8c3eb315a1.mov

### CODE REVIEW

**Review by rephus — APPROVED** [review]
Code looks good, I can see all variables in the new HTML. 

Have you tested this on the PR environment/ locally ? Do you know how ?

File: packages/backend/src/clients/EmailClient/templates/invitation.html [hunk 1]
```diff
@@ -871,9 +871,9 @@
                                                                                                         mso-line-height-rule: exactly;
                                                                                                         mso-text-raise: 8px;
                                                                                                     "
-                                                                                                        href="{{inviteUrl}}"
-                                                                                                        target="_blank"
-                                                                                                >Join
+                                                                                                    href="{{inviteUrl}}"
```
Comment (rephus) [inline packages/backend/src/clients/EmailClient/templates/invitation.html:874]: unrelated change ?

**Comment by yasmin-e** [comment]
> Code looks good, I can see all variables in the new HTML.
> 
> Have you tested this on the PR environment/ locally ? Do you know how ?

Tested locally! Waiting for Render to test in PR env :)

File: packages/backend/src/clients/EmailClient/templates/invitation.html (hunk 1 above)
Comment (yasmin-e) [reply packages/backend/src/clients/EmailClient/templates/invitation.html:874]: just ran `yarn workspace backend fix-format` and this happened
